CooperationHullMainSite/wwwroot/css/site.css

```diff
@@ -449,36 +516,31 @@ h1 {
     justify-content: center;
     gap: 2rem;
     flex-wrap: wrap-reverse;
-
     background-color: #23201E;
-
     position: relative;
     overflow: hidden;
-
     color: white;
     padding: 2rem;
     padding-bottom: 0;
 }
 
-.hww::before {
-    content: "";
-    position: absolute;
-    width: 100%;
-    height: 33%;
-    bottom: 0;
-
-    z-index: -1;
-
-    background-image: url("../assets/img/trees-pattern-grey.png");
-    background-repeat: no-repeat;
-}
+    .hww::before {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 33%;
+        bottom: 0;
+        z-index: -1;
+        background-image: url("../assets/img/trees-pattern-grey.png");
+        background-repeat: no-repeat;
+    }
 
-.hww .spot {
-    position: absolute;
-    right: 12%;
-    bottom: 0;
-    z-index: 0;
-}
+    .hww .spot {
+        position: absolute;
+        right: 12%;
+        bottom: 0;
+        z-index: 0;
+    }
```

Comment:
There's a lot of formatting changes like this in the PR, it would be good to move them into a separate one, if we feel strongly about it, to reduce noise in PRs that have more material changes (like this one!).

Separately, it might be worth getting a formatter set up so we don't have to worry about things like this?